### DISCUSSION TITLE
v2.16.60

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#5394] [General]
## What's Changed
* Fix: renderhook by @ThijmenKort in https://github.com/filamentphp/filament/pull/5374
* User Menu RTL aligment fix. by @mustafa-online in https://github.com/filamentphp/filament/pull/5383
* Section description compact by @howdu in https://github.com/filamentphp/filament/pull/5390

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.16.59...v2.16.60

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.16.60'>v2.16.60</a>.</em>

**Comment by romatexdev** [comment]
Hi Dan! After update my toggle column in tables dont work with error:
`Unable to call component method. Public method [setColumnValue] not found on component: [app.filament.resources.group-resource.pages.list-groups]`

The update is configured according to the documentation

I searched for changes in releases affected by this issue, but found nothing.

My group resource:
```
public static function table(Table $table): Table
    {
        return $table
            ->reorderable('order')
            ->columns([
                /////////////////////
                ToggleColumn::make('is_active')
                    ->sortable()
                    ->label(__('admin/fields.is_active')),
```
PHP 8.1
MariaDB version 10.6.11
composer show:
```
filament/filament                            v2.16.60 Effortlessly build TALL-powered admin pan...
filament/forms                               v2.16.60 Effortlessly build TALL-powered forms.
filament/notifications                       v2.16.60 Effortlessly build TALL-powered notificat...
filament/support                             v2.16.60 Associated helper methods and foundation ...
filament/tables                              v2.16.60 Effortlessly build TALL-powered tables.
laravel/framework                            v9.46.0  The Laravel Framework.
```

It feels like I'm missing something!

**Reply by danharrin** [reply]
Have you published the views? Delete those

**Reply by romatexdev** [reply]
That's right, Dan! I was simply using the blade views incorrectly. 
Thanks to your article,  I understood everything

https://filamentphp.com/blog/publishing-views-in-laravel

I'm sorry for this stupid question =)
